src/index.jsx

```diff
@@ -1,10 +1,11 @@
+import React from 'react'
 import core from './core'
 import TailSpin from './TailSpin'
 
 export default (
   ComposedComponent,
   {
-    LoadingIndicator = TailSpin,
-    ...rest,
+    renderLoadingIndicator = (() => <TailSpin />),
```

Comment:
To fit what was said in #36 this would become `({className, style}) => <TailSpin className={className} style={style} />`